rules/base-WINDOWS.ham

```diff
@@ -0,0 +1,68 @@
+# CLANG_EXE_BASE ?= "C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/Llvm/bin" ;
```

Comment:
Something like CLANG_EXE_BASE should be done in a `setup-toolset.sh`. See the `pierre-linux-clang` (PR #10) branch for how I've done that for Linux. In theory there should be much custom code for Clang on Windows.

The main issue should be to update the msvc_19_x64 toolset to have the Microsoft Clang binaries.